modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -958,7 +987,8 @@ public CompletableFuture<Void> upsertAll(Collection<BinaryRowEx> rows, InternalT
                 tx,
                 this::upsertAllInternal,
                 RowBatch::allResultFutures,
-                (res, req) -> false
+                (res, req) -> false,
+                false
```

Comment:
Currently retry logic enabled only for datastreamer as a showcase. All other places must be evaluated in a separate ticket due to a big amount of changes, because many tests can be affected.